pkgs/development/compilers/llvm/8/compiler-rt.nix

```diff
@@ -7,6 +7,10 @@ stdenv.mkDerivation {
   nativeBuildInputs = [ cmake python llvm ];
   buildInputs = stdenv.lib.optional stdenv.hostPlatform.isDarwin libcxxabi;
 
+  NIX_CFLAGS_COMPILE = [
+    "-DSCUDO_DEFAULT_OPTIONS=DeleteSizeMismatch=0:DeallocationTypeMismatch=0"
```

Comment:
I'd think `cmakeFlags` would be a more natural place to pass this? Or does that not work?

Reply:
I tried there and it ends up uninitialized